scripts/deploy/common.rb

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env ruby
 
+require 'open3'
+
+def fetch_password(password)
+    stdout, stderr, status = Open3.capture3("fetch-password --raw #{password}")
```

Comment:
This is likely to fail for a first time user. It would be nice to try/catch this and write out a nice error message on how to fix it.

Reply:
I'm hoping to do that as part of this jira: https://jira.corp.stripe.com/browse/MOBILESDK-2290

I'll try and fetch all the necessary passwords earlier in the script and have a try/catch with a nicer error message